Yafc.Parser/Data/FactorioDataDeserializer.cs

```diff
@@ -203,6 +203,9 @@ private void RenderIcons() {
             int rendered = 0;
 
             foreach (var o in allObjects) {
+                if (o.name.StartsWith("mining.")) {
+
+                }
```

Comment:
```suggestion
```

Reply:
oups, I shouldn't code in the middle of the night